crawler_detect/resources/known_crawler_useragents.txt

```diff
@@ -170,7 +170,7 @@ iqdb/0.1 (+http://iqdb.org/)
 Mozilla/5.0 (compatible; GimmeUSAbot/1.0; +https://gimmeusa.com/pages/crawler)
 Motoricerca-Robots.txt-Checker/1.0 (http://tool.motoricerca.info/robots-checker.phtml)
 Mozilla/5.0 (compatible; Tagoobot/3.0; +http://www.tagoo.ru)
-Mozilla/5.0 (Windows NT 6.1; WOW64; rv:26.0) Gecko/20100101 Firefox/26.0 Evidon (lab@evidon.com)
+Mozilla/5.0 (Windows NT 6.1; WOW64; rv:26.0) Gecko/20100101 Firefox
```

Comment:
Not a correct user agent it seems - 
Instead should be
Mozilla/5.0 (Windows NT 6.1; WOW64; rv:26.0) Gecko/20100101 Firefox/26.0